src/runtime/virtcontainers/qemu.go

```diff
@@ -113,6 +113,7 @@ const (
 	consoleSocket = "console.sock"
 	qmpSocket     = "qmp.sock"
 	vhostFSSocket = "vhost-fs.sock"
+	nydusdAPISock = "api.sock"
```

Comment:
`api.sock` is too generic that is hard to understand which component it's for.
How about changing to `nydusd.sock` or `nydusd-api.sock` ?

Reply:
I have updated to `nydusd-api.sock`